auction9-frontend/src/main.ts

```diff
@@ -4,9 +4,53 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
+import { Amplify } from '@aws-amplify/core';
+import { Auth } from '@aws-amplify/auth';
+
 if (environment.production) {
   enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(AppModule)
-  .catch(err => console.error(err));
+platformBrowserDynamic()
+  .bootstrapModule(AppModule)
+  .catch((err) => console.error(err));
+
+Amplify.configure({
+  Auth: {
+    // REQUIRED - Amazon Cognito Region
+    region: 'eu-west-1',
+
+    // OPTIONAL - Amazon Cognito User Pool ID
+    userPoolId: 'eu-west-1_hx2LFOcz2',
+
+    // OPTIONAL - Amazon Cognito Web Client ID (26-char alphanumeric string)
+    userPoolWebClientId: '6upsbt836sfi1kdigjvnp18mvr',
```

Comment:
@mjovanovic Should we put values like clientId and userPoolId in env variables and make it hidden from PR?